src/utils/contribution.js

```diff
@@ -269,18 +269,20 @@ export default class Contribution {
    * @param {*} value updated url of the definition
    * @returns a new object containing the schema for the sourceLocation
    */
-  static parseCoordinates(value) {
+  static validateAndCleanCoordinates(value) {
```

Comment:
I see now what this method is doing, it is really getting a "source location" object. It validates etc along the way. With that in mind I suggest `createSourceLocation` or some such as a name. If we had a `SourceLocation` class, we'd be calling a constructor or factory static here.

Note as well the method comment needs to be updated. the value param is not a URL any more.